.travis.yml

```diff
@@ -0,0 +1,19 @@
+# Config file for automatic testing at travis-ci.org
+sudo: false
+
+language: python
+python:
+  - 3.6
+  - 3.5
+
```

Comment:
I would recommend getting python 3.7 testing added. https://github.com/release-depot/atkinson/pull/2 has an example that appears to work. Python 3.7 is probably low risk at this point, but I know other projects has seen issues between 3.6 and 3.7 with what appear to be ABI breaks.

Reply:
I have no problem with that, but given that tests are passing right now, what would you think of merging this, and I will do that as my next patch?

Reply:
+1 from me. I will look for a followup patch.